code/game/objects/items/devices/radio/headset.dm

```diff
@@ -17,9 +32,21 @@
 
 /obj/item/radio/headset/examine(mob/user)
 	..()
-	to_chat(user, "<span class='notice'>To speak on the general radio frequency, use ; before speaking.</span>")
-	if (command)
-		to_chat(user, "<span class='notice'>Alt-click to toggle the high-volume mode.</span>")
+	
+	if(item_flags & IN_INVENTORY && loc == user)
+		// construction of frequency description
+		var/list/avail_chans = list("Use ; for the currently tuned frequency")
+		if(translate_binary)
+			avail_chans += "use :b for Binary"
+		if(length(channels))
+			for(var/i in 1 to length(channels))
+				avail_chans += "use [GLOB.channel_tokens[channels[i]]] for [lowertext(channels[i])]"

```

Comment:
Could probably also add :h description here.

Reply:
Resolved in c280256